CONTRIBUTING.md

```diff
@@ -70,6 +70,25 @@ contents of the `web-design-standards` project under `node_modules/uswds` within
 the `project-using-npm`. For more information on `npm-link`, you can read about
 it in the [`npm-link` documentation](https://docs.npmjs.com/cli/link).
 
+## Local Gem Development for [Styleguide Website](https://playbook.cio.gov/designstandards)
```

Comment:
I can update this to say `Playbook website` / `documentation website`. I'll wait for @kategarklavs to weigh in as well.